standard/date_ext.lua

```diff
@@ -22,7 +22,7 @@ DateExt.minTimestamp = -62167219200
 -- 9999-12-31 23:59:59
 DateExt.maxTimestamp = 253402300799
 
--- 1970-01-01 00:00:00
+-- 0000-01-01 00:00:00
 DateExt.epochZero = 0
```

Comment:
it is still 1970 here

Reply:
I rather change the usage to `minTimestamp` instead, or rename this is `defaultTime` (or smt similar)

Reply:
yeah usage gets switched to `minTimestamp`

Reply:
Oh for sure, but it will be removed later ;)
I think I'd like to keep minTimestamp and the default time as two sep variables, as that will simplify if we need to change in the future.

Reply:
okay for me^^